Vonk.Plugin.DocumentOperation/DocumentService.cs

```diff
@@ -269,12 +269,12 @@ private IssueComponent ReferenceNotResolvedIssue(string failedReference, bool mi
             if (missingReferenceIsLocal)
             {
                 issue.Code = IssueType.NotFound;
-                issue.Details = new CodeableConcept("http://hl7.org/fhir/ValueSet/operation-outcome", "MSG_LOCAL_FAIL", "Unable to resolve local reference to resource " + failedReference);
+                issue.Details = new CodeableConcept("http://vonk.fire.ly/fhir/ValueSet/OperationOutcomeIssueDetails", "MSG_LOCAL_FAIL", "Unable to resolve local reference to resource " + failedReference);
```

Comment:
MSG_LOCAL_FAIL is part of the "normal" valueset of operation-outcome, right? So we can just use that code from that system, or not?

Reply:
Resolved by agreeing that we stick to a single ValueSet.